### ISSUE TITLE
Change locale, timezone and so

### ISSUE THREAD

**Issue by jiauka** [rithvikvibhu/GHLocalApi#41]
name, locale, timezone can be changed by:

`curl -X POST -H "Content-Type: application/json" -d '{"name": "myChromecast","settings":{"timezone":"Europe\/Madrid","time_format":2,"locale":"es"},"opt_in":{"stats":false}}' http://10.10.0.10:8008/setup/set_eureka_info
`

**Comment by rithvikvibhu** [comment]
Most of the fields from `eureka_info` are editable with `set_eureka_info`. But this could be mentioned along with viewing locale and timezone. Will add a note there, then close the issue.

**Comment by rithvikvibhu** [comment]
Will add this with the documentation overhaul.

**Comment by darkzbaron** [comment]
Any idea as to how to actually change the language, it does not work when using the above mentionned method!

**Comment by rithvikvibhu** [comment]
@darkzbaron have you seen the new authentication process?
https://rithvikvibhu.github.io/GHLocalApi/#section/Google-Home-Local-API/Authentication

**Comment by darkzbaron** [comment]
Yes, I am running a post request on `set_eureka_info` , it returns 200 and set the value but the Google Home does not change language. When I change with the app , howerver, this works

**Comment by rithvikvibhu** [comment]
Ah. Maybe it's one of those settings that the app sends to Google first, which is then reflected on device. I don't have that device with me right now to test. If you want to give it a try, there's a how-to mitm with grpc in #39.

**Comment by darkzbaron** [comment]
Cool, I'll do that this week end and revert, thanks so much for the help!
